azurerm/helpers/validate/cosmos.go

```diff
@@ -42,3 +42,24 @@ func CosmosThroughput(v interface{}, k string) (warnings []string, errors []erro
 
 	return warnings, errors
 }
+
+func CosmosMaxThroughput(v interface{}, k string) (warnings []string, errors []error) {
+	value := v.(int)
+
+	if value < 4000 {
```

Comment:
Can we switch all of the `value` variables to be `v` as well?

```suggestion
	if v < 4000 {
```